tests/map.spec.js

```diff
@@ -37,11 +37,13 @@ describe('map usage', () => {
   it('onFieldsChange works', () => {
     const Test = createForm({
       withRef: true,
-      onFieldsChange(props, fields) {
+      onFieldsChange(props, fields, allFields) {
```

Comment:
You can get fields from this `props`，for you had pass fields to upper component by `onChange`